custom/enikshay/two_b_release_1/management/commands/enikshay_2b_case_properties.py

```diff
@@ -0,0 +1,496 @@
+                "update": props,
+            },
+        )
+
+    def migrate_occurrence(self, occurrence, episodes):
+        self.total_occurrences += 1
+        episodes = [(case.opened_on, case) for case in episodes
+                    if case.type == CASE_TYPE_EPISODE
+                    and any([index.referenced_id == occurrence.case_id for index in case.indices])
+                    and case.dynamic_case_properties().get('episode_type') == "confirmed_tb"]
+        if not episodes:
+            return None
+
+        episode = max(episodes)[1]  # Most recently opened episode for the occurrence
+
+        props = {
+            'current_episode_type': episode.get_case_property('episode_type'),
+            'disease_classification': episode.get_case_property('disease_classification'),
+            'site_choice': episode.get_case_property('site_choice'),
+            'site_detail': episode.get_case_property('site_detail'),
+            'key_population_status': episode.get_case_property('key_population_status'),
+            'key_populations': episode.get_case_property('key_populations'),
+        }
+
+        return CaseStructure(
+            case_id=occurrence.case_id,
+            walk_related=False,
+            attrs={
+                "create": False,
+                "update": props,
+            },
+        )
+
+    def migrate_episode(self, episode, episodes):
+        self.total_episodes += 0
+
+        def is_episode_of_occurrence(episode, occurrence_id):
+            return any(index.referenced_id == occurrence_id for index in episode.indices)
+
+        occurrence_ids = [index.referenced_id for index in episode.indices
+                          if index.referenced_type == CASE_TYPE_OCCURRENCE]
+        occurrence_id = occurrence_ids[0] if occurrence_ids else None
+
+        latest_episode_id = max((case.opened_on, case.case_id) for case in episodes
+                                if is_episode_of_occurrence(case, occurrence_id)
+                                and not case.closed)[1]
+
+        test_type = episode.get_case_property('test_confirming_diagnosis')
+        props = {
+            'is_active': 'yes' if episode.case_id == latest_episode_id else 'no',
+            'dosage_display': episode.get_case_property('full_dosage'),
+            'dosage_summary': episode.get_case_property('full_dosage'),
+            'rft_general': 'diagnosis_dstb',
+            'diagnosis_test_type': test_type,
+            'diagnosis_test_type_label': TEST_TO_LABEL.get(test_type, ""),
+        }
+
+        treatment_status = episode.get_case_property('treatment_status')
+        if treatment_status == 'second_line_treatment':
+            props['treatment_status'] = 'initiated_second_line_treatment'
+        elif treatment_status == 'yes_phi':
+            props['treatment_status'] = 'initiated_first_line_treatment'
+        elif treatment_status == 'yes_private':
+            props['treatment_status'] = 'initiated_outside_facility'
```

Comment:
It looks like `treatment_status` is used in reports in some places. Is this something that is expected? Could be an error in the datasource: https://github.com/dimagi/commcare-hq/blob/master/custom/enikshay/ucr/data_sources/episode.json#L5880-L5888, but otherwise I think you might be overwriting that property here.